third_party/2and3/toml.pyi

```diff
@@ -0,0 +1,14 @@
+from typing import Any, IO, Mapping, MutableMapping, Optional, Type, Union
+import datetime
+
+
+class TomlDecodeError(Exception): ...
+
+class TomlTz(datetime.tzinfo):
+    def __init__(self, toml_offset: str) -> None: ...
+
+def load(f: Union[str, list, IO[str]], _dict: Type[MutableMapping[str, Any]] = ...) -> MutableMapping[str, Any]: ...
+def loads(s: str, _dict: Type[MutableMapping[str, Any]] = ...) -> MutableMapping[str, Any]: ...
+
+def dump(o: Mapping[str, Any], f: IO[str]) -> str: ...
```

Comment:
`f` can actually be any object with a `write` method. Perhaps we should go with the python/typing#564 of defining a simple protocol for this argument. In this case, it would just have `def write(self, obj: str) -> Any: ...`.